ui/src/store/useSituationsStore.ts

```diff
@@ -0,0 +1,26 @@
+import { defineStore } from 'pinia'
+import { getSituations } from '@/services/AlarmService'
+import { TSituation } from '@/types/TSituation'
+
+type TState = {
+	situations: TSituation[]
+}
+
+export const useSituationsStore = defineStore('situationsStore', {
+	state: (): TState => ({
+		situations: []
+	}),
+	actions: {
+		async getSituations() {
+			getSituations().then((result) => {
+				if (result) {
+					this.situations = processAlarmList(result.alarm)
+				}
+			})
+		}
```

Comment:
If you want to use `async`, you should just be able to do:

```
		async getSituations() {
			const result = await getSituations()
			if (result) this.situations = processAlarmList(result.alarm)
		}
```

But if you want to use the older `.then((result) => ...)` syntax instead, then you should be able to remove the `async` keyword